grpc/src/compiler/go_generator.cc

```diff
@@ -70,12 +70,11 @@ void GenerateImports(grpc_generator::File *file, grpc_generator::Printer *printe
 	printer->Print("//If you make any local changes, they will be lost\n");
```

Comment:
this file is shared with the GRPC project, so formatting changes here are not desirable, they will make future updates/merges much harder.